lib/manageiq/api/common/rbac/access.rb

```diff
@@ -0,0 +1,72 @@
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class Access
+          attr_reader :acl
+          DEFAULT_LIMIT = 500
+          def initialize(resource, verb)
+            @resource = resource
+            @verb     = verb
+            @regexp   = Regexp.new(":(#{@resource}|\\*):(#{@verb}|\\*)")
+            @app_name = ENV["APP_NAME"]
+          end
+
+          def process
+            ManageIQ::API::Common::RBAC::Service.call(RBACApiClient::AccessApi) do |api|
+              Rails.logger.info("Fetch access list for #{@app_name}")
+              @acl = ManageIQ::API::Common::RBAC::Service.paginate(api, :get_principal_access, {:limit => DEFAULT_LIMIT}, @app_name).select do |item|
+                Rails.logger.info("Found ACL: #{item}")
+                @regexp.match(item.permission)
+              end
+            end
+            self
+          end
+
+          def accessible?
+            Rails.logger.info("ACL for #{@app_name} #{@acl}")
```

Comment:
This log message is strange as it just lists the ACLs but every time you call this method?

Reply:
Removed